docs/featureflags.md

```diff
@@ -0,0 +1,89 @@
+# Agent feature flags
+## Overview
+The feature flags can adjust behavior in specific cases
+
+## Usage
+To use the flag in the pipeline you need to [define variable](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables) with specific name. 
+
+Please note that "Environment name" column may differ from "Template name".
```

Comment:
Suggest adding some reference to 'runtime' so the reader knosw that templatename == runtimevariable